addons/binding/org.openhab.binding.mysensors.test/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.openhab.binding</groupId>
+		<artifactId>pom</artifactId>
+		<version>2.2.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>org.openhab.binding.mysensors.test</artifactId>
+	
+	<name>MySensors.test Binding</name>
+	<packaging>eclipse-plugin</packaging>
+    
+    <profiles>
```

Comment:
I don't know, I took it from here: https://github.com/openhab/static-code-analysis/blob/master/docs/maven-plugin.md